spec/test-utils/webrtc.ts

```diff
@@ -150,7 +150,12 @@ export class MockRTCPeerConnection {
         this.localDescription = {
             sdp: DUMMY_SDP,
             type: "offer",
-            toJSON: function () {},
+            toJSON: function (): RTCSessionDescriptionInit {
+                return {
+                    sdp: DUMMY_SDP,
+                    type: "offer",
+                };
+            },
         };
```

Comment:
Optional suggestion to avoid repetition:
```typescript
        const localDescriptionJSON = {
            sdp: DUMMY_SDP,
            type: "offer" as RTCSdpType,
        };
        this.localDescription = {
            toJSON: () => localDescriptionJSON,
            ...localDescriptionJSON,
        };
```